pepper-apis/src/test/java/org/broadinstitute/ddp/db/dao/QuestionDaoTest.java

```diff
@@ -1133,60 +952,6 @@ public void testGetPicklistQuestion_success() {
         });
     }
 
-    @Test
-    public void testGetPicklistQuestion_withGroups() {
```

Comment:
For this and the "text question" tests below, do we have equivalent tests elsewhere? Looks like they're not just testing the removed methods. They're creating picklist/text questions and making we can fetch them. Have we sufficiently covered these in other tests?

Reply:
> For this and the "text question" tests below, do we have equivalent tests elsewhere? Looks like they're not just testing the removed methods. They're creating picklist/text questions and making we can fetch them. Have we sufficiently covered these in other tests?

So it needs to analyze more deeply the purpose of tests which use these methods.. 
I will try to do it.

Reply:
I mean, I don't see any more "deep purpose" with these tests. They just create a question, insert it, and make sure we can read it back out. I'm sure we should have other tests that does this already.